admin/proc-install-operator.adoc

```diff
@@ -0,0 +1,35 @@
+[id='proc-install-operator_{context}']
+
+= Installing the {product} Operator
+
+As an administrator, you can install the {product} Operator. Authorized users can use the Operator to install {product} on the following platforms:
+
+* {ocp-brand-name} ({ocp-very-short})
+* {eks-brand-name} ({eks-short})
+* {aks-brand-name} ({aks-short})
+
+.Prerequisites
+
+* You are logged in as an administrator on the {ocp-short} web console.
+* You have configured the appropriate roles and permissions within your project to create an application. For more information, see the link:https://docs.openshift.com/container-platform/4.14/applications/index.html[Red Hat OpenShift documentation on Building applications].
+
+.Procedure
+
+. In the *Administrator* perspective of the {ocp-short} web console, navigate to *Operators* > *OperatorHub*.
+
+. In the *Filter by keyword* box, enter {product-short} and click the *{product}* Operator card.
+
+. On the *{product} Operator* page, click *Install*.
+
+. On the *Install Operator* page, click *Install*.
```

Comment:
I would add something here about which Channel to select for the Operator. I went through the installation today and there is one called Fast and one called Fast-1.1 I think, and I wasn't sure what these meant or the difference between them. Would be good to clarify this I think.

Reply:
GTK. Let's definitely open a separate ticket for that, since the main goal of this PR is to restructure the existing content and relocate all installation procedures to the Admin guide.

Reply:
There is a JIRA for this already: https://issues.redhat.com/browse/RHIDP-1959

Reply:
Looks like a ticket was already opened for this `Fast`/`Fast-1.1` issue https://issues.redhat.com/browse/RHIDP-1959